src/core/constants/AVERAGE_BLOCKS_PER_10_MINS.ts

```diff
@@ -0,0 +1,11 @@
+import { AVERAGE_BLOCKS_PER_DAY } from './AVERAGE_BLOCKS_PER_DAY'
+import { Chain } from './chains'
+
+export const AVERAGE_BLOCKS_PER_10_MINUTES: Record<Chain, number> =
+  Object.entries(AVERAGE_BLOCKS_PER_DAY).reduce(
+    (acc, [chain, blocksPerDay]) => {
+      acc[chain as unknown as Chain] = (blocksPerDay * 10) / (24 * 60)
```

Comment:
since object keys are strings, indexing by "Chain" is a little annoying

Reply:
In other places of the code I have used `+chain as Chain` so I can compare it and pass the correct type down the line, but in this case it's going to convert them to a string for the key, so it doesn't matter.